app/api/helpers/system_mails.py

```diff
@@ -261,5 +261,15 @@
         'message': (
             u"This is a  <strong> Test </strong> E-mail."
         )
+    },
+    PENDING_ORDER_EXPIRED: {
```

Comment:
@uds5501 this issue is supposed to send an email and notification to user using which he/she can deactivate the expired order. So this issue involves an endpoint maybe where user can access his/her expired ticket within 24 hours and reactivate it and complete order by paying.
So maybe you can proceed by understanding these points -
 - User clicks on the link and is redirected to a page of his/her expired ticket (within 24 hours). 
 - The status is changed from `expired` to `pending` when user uses the link
 - The user can pay using the payment methods available on the `pending order page`.
 - If user fails to complete payment 2nd time, the ticket should be expired and then this normal mail should be sent out to him that he/she can order by visiting tickets page of the event

Reply:
@shreyanshdwivedi Okay, So here are a few things which I will add, please let me know if they confirm to the pattern required in this refactor.

- The expiry mail will only be sent if there are any tickets left for that particular event.
- If user uses the link, the order status will be set to `pending`



> * If user fails to complete payment 2nd time, the ticket should be expired and then this normal mail should be sent out to him that he/she can order by visiting tickets page of the event

I am just no sure how to implement this. Any ideas?

Reply:
Corresponding to your first point - As user has pending order, the tickets are already reseved. No need to check if ticket exists while sending out mail for expired order.
When a user's pending ticket expires normally send him a mail with a link using which he/she can access that expired ticket and update it's status to pending again.
You can take example of reset password where email is send out alongwith a link and user accesses a page via link to update password. Try to create a endpoint to handle the click on link

Reply:
@shreyanshdwivedi Understood.